components/title/basic/src/index.js

```diff
@@ -0,0 +1,52 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+export default function TitleBasic ({align, className, subtitle, title}) {
+  function buildAlignClassName (alignProp) {
```

Comment:
since this is a private function, i would define this function as `_buildAlignClassName`